src/frontend/src/handler/create_table.rs

```diff
@@ -178,22 +162,137 @@ pub fn bind_sql_columns(
             name: name.real_value(),
             field_descs,
             type_name: "".to_string(),
+            generated_column: None,
         });
     }
 
-    Ok((column_descs, pk_column_id))
+    Ok(column_descs)
+}
+
+fn check_generated_column_constraints(
+    column_name: &String,
+    expr: &ExprImpl,
+    column_catalogs: &[ColumnCatalog],
+    generated_column_names: &[String],
+) -> Result<()> {
```

Comment:
What will happen if the data type of `expr` doesn't match the column definition?

Reply:
I noted this in the tracking issue. In this case, a `cast_assign` is needed.

Reply:
Then should we check whether the type matches currently?